src/library_egl.js

```diff
@@ -319,25 +324,28 @@ var LibraryEGL = {
       }
       contextAttribs += 8;
     }
+#if USE_WEBGL2
+    if (glesContextVersion < 2 || glesContextVersion > 3) {
```

Comment:
how does `< 2 || > 3` differ from `!= 2`?

Reply:
It allows 3 now.